configure.sh

```diff
@@ -214,6 +218,67 @@ if [ -z "${QUAY__DOCKERCONFIGJSON}" ]; then
     BYPASS_OPTIONAL_INPUT+=",QUAY__DOCKERCONFIGJSON"
 fi
 
+# Configure remote clusters
```

Comment:
I would say also gate all this logic behind a flag like `ENABLE_REMOTE_CLUSTER_SETUP`? It can be false by default

or maybe gate it with a question like would you like to enable remote cluster setup? Y/N

Maybe I don't want to do any remote cluster setup, so I dont want to answer these questions.. 🤔